index.js

```diff
@@ -11,6 +11,7 @@ function CompressionPlugin(options) {
 	options = options || {};
 	this.asset = options.asset || "[path].gz[query]";
 	this.algorithm = options.algorithm || "gzip";
+	this.changeNewFileName = options.changeNewFileName || false;
```

Comment:
@michael-ciniawsky I didn't see any tests for the existing code.
I would like to add to them if they exist.
Do you have any tests that you already run on the code?